src/VirtoCommerce.CatalogModule.Data/Search/Indexing/CategoryDocumentBuilder.cs

```diff
@@ -86,6 +87,11 @@ protected virtual IndexDocument CreateDocument(Category category)
                 document.Add(new IndexDocumentField("__outline", outline.ToLowerInvariant()) { IsRetrievable = true, IsFilterable = true, IsCollection = true });
             }
 
+            foreach (var outlineItem in GetOutlineStrings(category.Outlines, getNameLatestItem: true))
+            {
+                document.Add(new IndexDocumentField($"__outlines", outlineItem) { IsRetrievable = true, IsFilterable = true, IsCollection = true });
```

Comment:
Maybe we could call it:
"__outline_named"
To distinguish from existing `"__outline"` property

Reply:
have added outline delimiter

Reply:
The name `"__outlines"` is still not clear for me, because I do not see a proper semantic difference between `"__outlines"` and `"__outline"` based just on the name. It could be that for `"__outlines"` there are multiple outlines, and for `"__outline"` there is only one. But it is not our case. So maybe `"__outline_named"` would be better

Reply:
agree